corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -58,7 +58,7 @@
     <% if (actions || isMultiSelect) { %>
     <div class="case-list-actions">
       <% if (isMultiSelect) { %>
-      <button type="button" class="btn btn-success btn-lg formplayer-request" id="multi-select-continue-btn">{% trans "Continue" %}</button>
+      <button type="button" class="btn btn-success btn-lg formplayer-request" id="multi-select-continue-btn">{% trans "Continue (" %}<%- selectedCaseIdCount %>{% trans ")" %}</button>
```

Comment:
Just translate "Continue" - there's no need to translate the parens, and they'll make the string more weird/confusing for the translators.